docs/source/_static/custom.css

```diff
@@ -10,3 +10,21 @@ div.wy-side-nav-search {
   margin: auto;
   display: block;
 }
+
+/* Elevation
+  *
+  * We style box-shadows using Material Design's idea of elevation. These particular numbers are taken from here:
+  *
+  * https://github.com/material-components/material-components-web
+  * https://material-components-web.appspot.com/elevation.html
+  */
+.shadow {
+  border: 1px solid #E0E0E0;
+  box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2),
```

Comment:
I prefer something more like 8. These have very different content from cards, and the edges get very busy with content/border/shadow. I would probably remove the border, and do an 8 elevation.

Reply:
I met with @jzf2101 and @tgeorgeux on appear.in and we discussed removing the border and using the elevation 8 shadow.